### DISCUSSION TITLE
Scrutinio trimestre

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#615] [Richieste di aiuto]
Salve io quando apro lo scrutinio mi risulta "ERROR 500" .So che questo aggiornamento ha reso impossibile aprire lo scrutinio ma entro quando sarà risolto? Così che si recuperano.

**Comment by trinko** [comment]
[vedi](https://github.com/iisgiua/giuaschool/issues/612#issuecomment-1861221063)

**Comment by gianlucaaiello56** [comment]
va bene ,quindi è sicuro che a gennaio sarà possibile recuperare gli scrutini giusto?

**Comment by trinko** [comment]
Sarà sistemato al massimo entro l'ultima settimana di gennaio.
